LiteDB/Engine/Services/CacheService.cs

```diff
@@ -1,11 +1,18 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading;
+
+#if !NET35
```

Comment:
LiteDB doesn't support NET35, so these conditions could be removed here and below.

Reply:
this fixes v4 which targets 3.5